python/glutil.cpp

```diff
@@ -147,7 +147,15 @@ void register_glutil(py::module &m) {
     #define C(name) gl.attr(#name) = py::int_(GL_##name);
     py::module gl = m.def_submodule("gl");
 
+    gl.def("glEnable", [](py::int_ cap) {
+              glEnable(cap);
+           }, py::arg("cap"));
+    gl.def("glDisable", [](py::int_ cap) {
+              glDisable(cap);
+           }, py::arg("cap"));
```

Comment:
Yep, that looks good. You can also replace ``py::int_`` by ``GLenum`` in these two definitions, which is the right type in this case.